files/build_templates/docker_image_ctl.j2

```diff
@@ -100,6 +105,7 @@ start() {
     # Obtain our HWSKU as we will mount directories with these names in each docker
     HWSKU=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["hwsku"]'`
     {%- endif %}
+    HOSTNAME=`sonic-cfggen -d -v 'DEVICE_METADATA["localhost"]["hostname"]'`
```

Comment:
>-d [](start = 27, length = 2)

-d will check Redis DB, which is inside a container. Can we just check -m Minigraph for less dependency?

Reply:
done

Reply:
> Can we just check -m Minigraph for less dependency?

One concern I have with this approach is what if the minigraph on the device differs from the current config in the DB?